ci/prow/config.yaml

```diff
@@ -2148,7 +2148,26 @@ periodics:
       - "--profile-name=coverage_profile.txt"
       - "--cov-target=."
       - "--cov-threshold-percentage=80"
-
+- cron: "0 19 * * 1" # Run at 10:00PST/11:00PST every Monday (19:00 UTC)
+  name: cleanup-old-images
```

Comment:
To keep the naming pattern, I suggest using `ci-knative-cleanup`.

Reply:
done